frontend-html/src/gui/Components/ScreenElements/Table/Header.tsx

```diff
@@ -23,6 +23,7 @@ import S from "./Header.module.scss";
 import { IOrderByDirection } from "model/entities/types/IOrderingConfiguration";
 import { Draggable, DraggableProvided } from "react-beautiful-dnd";
 import { action, observable } from "mobx";
+import cx from "classnames";
```

Comment:
So what does `cx` mean? :-)

Reply:
`cx` is a function from the `classnames` package. The function joins css classes